2020/hamming-codes/spanish/sentence_translations.json

```
@@ -1388,7 +1388,7 @@
  {
   "input": "And now that you know how to do all this by hand, I'd like to show you how you can carry out the core part of all of this logic with a single line of Python code.",
   "model": "nmt",
-  "translatedText": "Y ahora que sabes cómo hacer todo esto a mano, me gustaría mostrarte cómo puedes llevar a cabo la parte central de toda esta lógica con una sola línea de código Python.",
+  "translatedText": "Y ahora que sabes cómo hacer todo esto a mano, me gustaría mostrarte cómo puedes llevar a cabo la parte central de toda esta lógica con una sola línea de código de Python.",
```

```suggestion
  "translatedText": "Y ahora que sabes cómo hacer todo esto a mano, me gustaría mostrarte cómo puedes llevar a cabo la parte central de toda esta lógica con una sola línea de código Python.",
```
or even
```suggestion
  "translatedText": "Y ahora que sabes cómo hacer todo esto a mano, me gustaría mostrarte cómo puedes llevar a cabo la parte central de toda esta lógica con una sola línea de código en Python.",
```
but the `de` seems weird to me, might be an Spaniard thing.

---

I really didn't like it at first, I think using "en" is better